lib/csvProcessor.js

```diff
@@ -98,6 +98,7 @@ CSVProcessor.prototype.individual = function() {
                 }
                 contextList.push(this.fileSpec);
                 if (errors[id].scope === 'hmda') {
+                    contextList.push(this.fileSpec.hmdaFile);
```

Comment:
What does this change have to do with the rest of the fix? Seems like a stray change made it into your commit